apollo-router/src/plugins/telemetry/tracing/mod.rs

```diff
@@ -0,0 +1,28 @@
+pub mod apollo;
+pub mod apollo_telemetry;
+pub mod datadog;
+pub mod jaeger;
+#[cfg(any(feature = "otlp-grpc", feature = "otlp-http"))]
+pub mod otlp;
+pub mod zipkin;
+
+#[cfg(test)]
+mod test {
+    use tracing::{info_span, Instrument};
+
+    pub async fn run_query() {
+        let span = info_span!("client_request");
+        let client = reqwest_middleware::ClientBuilder::new(reqwest::Client::new())
```

Comment:
could this test be done with setting the opentelemetry headers manually, to avoid bringing back reqwest-middleware and reqwest-tracing?

Reply:
Fixed in [bf577ae](https://github.com/apollographql/router/pull/782/commits/bf577ae79247009dd0c2e6face82f240d1612fdc)